### PR TITLE
Update BCD info, part 4

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #19185 

The PR updates BCD info in WebAPI docs.

Note: It's easy to review the changes. Open the preview URLs hosted by the PR companion, in new tabs, and look at the BCD tables at the bottom of the pages.

### CODE REVIEW

File: files/en-us/web/api/cookiestore/delete/index.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ tags:
   - API
   - Method
   - Reference
+  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/cookiestore/delete/index.md:9]: The tag is already listed.
```suggestion
```

File: files/en-us/web/api/cookiestore/get/index.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ tags:
   - API
   - Method
   - Reference
+  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/cookiestore/get/index.md:9]:
```suggestion
```

File: files/en-us/web/api/cookiestore/getall/index.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ tags:
   - API
   - Method
   - Reference
+  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/cookiestore/getall/index.md:9]:
```suggestion
```

File: files/en-us/web/api/cookiestore/set/index.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ tags:
   - API
   - Method
   - Reference
+  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/cookiestore/set/index.md:9]:
```suggestion
```

File: files/en-us/web/api/css_properties_and_values_api/index.md [hunk 5]
```diff
@@ -4,6 +4,7 @@ slug: Web/API/CSS_Properties_and_Values_API
 page-type: web-api-overview
 tags:
   - Houdini
+  - Experimental
 browser-compat: api.CSS.registerProperty
 ---
 {{SeeCompatTable}} The **CSS Properties and Values API** — part of the [CSS Houdini](/en-US/docs/Web/Guide/Houdini) umbrella of APIs — allows developers to explicitly define their {{cssxref('--*', 'CSS custom properties')}}, allowing for property type checking, default values, and properties that do or do not inherit their value.
```
Comment (teoli2003) [inline files/en-us/web/api/css_properties_and_values_api/index.md:10]: Note that there should be a sidebar here. We likely need a new entry under `GroupData.json` for this API. (Follow-up PR?)
```suggestion
{{SeeCompatTable}}
The **CSS Properties and Values API** — part of the [CSS Houdini](/en-US/docs/Web/Guide/Houdini) umbrella of APIs — allows developers to explicitly define their {{cssxref('--*', 'CSS custom properties')}}, allowing for property type checking, default values, and properties that do or do not inherit their value.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
Thanks!

> Note that there should be a sidebar here. We likely need a new entry under `GroupData.json` for this API. (Follow-up PR?)
I'll create the PR for /en-us/web/api/css_properties_and_values_api

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**